source/ui/BioXAS/BioXASZebraLogicBlockInputInvertedView.cpp

```diff
@@ -0,0 +1,77 @@
+#include "BioXASZebraLogicBlockInputInvertedView.h"
+
+BioXASZebraLogicBlockInputInvertedView::BioXASZebraLogicBlockInputInvertedView(BioXASZebraLogicBlockInput *control, QWidget *parent) :
```

Comment:
I think @XuetaoLiu-CLS is right that we could probably generalize these.  Since you've made the ZebraInput class answer both `isEnabled()` and `isInverted()` questions, you should be able to make a view that updates its appearance  based on those questions being answered (and correspondingly, their signals being emitted).